### PR TITLE
formats prefix to not be flowrun every time

### LINKED ISSUE #402: Change logging prefix to avoid unnecessary task definition registrations
The current implementation generates an `awslogs-prefix` for every flow run, creating an unnecessary amount of task definition registrations which could lead to rate limits.

### PR DESCRIPTION
**Pull request by jeanluciano**
Closes #402 

### Checklist

- [ x ] References any related issue by including "Closes #<Issue Number>" or "Closes <Issue URL>".
  - If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect-aws/issues/new/choose) first.
- [ x ] Includes tests or only affects documentation.
- [ x ] Passes `pre-commit` checks.
  - Run `pre-commit install && pre-commit run --all` locally for formatting and linting.
- [ ] Includes screenshots of documentation updates.
  - Run `mkdocs serve` view documentation locally.

### CODE REVIEW

File: prefect_aws/workers/ecs_worker.py [hunk 1]
```diff
@@ -507,6 +508,14 @@ class ECSVariables(BaseVariables):
             " for available options. "
         ),
     )
+    cloudwatch_logs_prefix: Optional[str] = Field(
+        default=None,
+        description=(
+            "When `configure_cloudwatch_logs` is enabled, this setting may be used to"
+            " set a prefix for the log group. If not provided, the default prefix will"
+            " be `workpool_name_flow-run`."
```
Comment (kevingrismore) [inline prefect_aws/workers/ecs_worker.py:516]:
```suggestion
            " be `prefect-logs_<work_pool_name>_<deployment_id>`."
```

File: prefect_aws/workers/ecs_worker.py [hunk 2]
```diff
@@ -507,6 +508,14 @@ class ECSVariables(BaseVariables):
             " for available options. "
         ),
     )
+    cloudwatch_logs_prefix: Optional[str] = Field(
+        default=None,
+        description=(
+            "When `configure_cloudwatch_logs` is enabled, this setting may be used to"
+            " set a prefix for the log group. If not provided, the default prefix will"
+            " be `prefect-logs_<work_pool_name>_<deployment_id>`."
```
Comment (kevingrismore) [inline prefect_aws/workers/ecs_worker.py:516]:
```suggestion
            " be `prefect-logs_<work_pool_name>_<deployment_id>`. If `awslogs-stream-prefix`"
            " is present in `Cloudwatch logs options` this setting will be ignored."
```

**Review by kevingrismore — COMMENTED** [review]
Just so it's clear to users what happens if both are set.

File: tests/test_ecs.py [hunk 3]
```diff
@@ -1245,6 +1245,7 @@ async def test_cloudwatch_log_options(aws_credentials):
         if container["name"] == "prefect":
             # Assert that the 'prefect' container has logging configured with user
             # provided options
+            print(container["logConfiguration"])
```
Comment (desertaxle) [inline tests/test_ecs.py:1248]:
```suggestion
```

**Review by desertaxle — APPROVED** [review]

**Pull request merged**